src/backend/distributed/planner/modify_planner.c

```diff
@@ -102,7 +102,6 @@ ErrorIfQueryNotSupported(Query *queryTree)
 	Oid distributedTableId = ExtractFirstDistributedTableId(queryTree);
 	uint32 rangeTableId = 1;
 	Var *partitionColumn = PartitionColumn(distributedTableId, rangeTableId);
-	char partitionMethod = PartitionMethod(distributedTableId);
```

Comment:
No, it is not, but this is a very small change which removes an unnecessary warning and nobody fixed for a while. Therefore, I thought that I should fix it.